src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/AgoraMongoDao.scala

```diff
@@ -83,6 +76,33 @@ class AgoraMongoDao(collection: MongoCollection) extends AgoraDao {
     }
   }
 
+  override def find(entity: AgoraEntity, projection: Option[AgoraEntityProjection]): Seq[AgoraEntity] = {
+    projection match {
+      case Some(project) => find(EntityToMongoDbObject(entity), projection)
```

Comment:
Should use project and not projection (doesn't really make a diff, I think - but that's what you do elsewhere).